File: 1]. DSA + CP/1]. DSA/1]. Data Structures/01]. Array/Python/_001)_Search_Element.py
# function to perform linear search for element in array
def search(arr, element):
    for index in range(0, len(arr)):
        if arr[index] == element:
            return index

    # if no such number exist in the array return -1
    return -1


arr = [20, 5, 7, 25]
element = 5

print("Array Searched:", arr)
print("Searching for:", element)
print("Searched index=", search(arr, element))

File: 1]. DSA + CP/1]. DSA/1]. Data Structures/01]. Array/JavaScript/_001)_Search_Element.js
// function to search for element in array
function search(arr, x) {
    for (let index = 0; index < arr.length; index++) {
        if (arr[index] === x) {
            return index;
        }
    }
    //if no such number exist in the array return -1
    return -1;
}

const arr = [20, 5, 7, 25];
console.log(`Searched index= ${search(arr, 5)}`);
